src/petals/utils/packaging.py

```diff
@@ -0,0 +1,50 @@
+import re
+from typing import Any
+
+import torch
+from hivemind import nested_flatten, nested_pack
+
+# TODO: Move functions to hivemind
+
+
+def _mark_masked_tensor(index: int):
+    return b"__T" + str(index).encode()
+
+
+def _is_masked_tensor(item: Any):
+    return isinstance(item, bytes) and re.match(b"^__T\d+$", item) is not None
```

Comment:
```suggestion
    return isinstance(item, bytes) and item.startswith("__T") is not None
```

I strongly recommend to avoid `re` to increase performance. If you keep it, please use `re.fullmatch(rb"__T\d+", item)` instead (does the same thing but won't fail if a user adds new `\smth` sequence).